### PR TITLE
Updates Openly org name to reflect openly-engineering

### PR DESCRIPTION
**Pull request by mwielbut**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
-	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/golangci/revgrep v0.0.0-20180812185044-276a5c0a1039
-	github.com/google/uuid v1.1.1 // indirect
-	github.com/huandu/xstrings v1.3.0 // indirect
-	github.com/imdario/mergo v0.3.8 // indirect
-	github.com/mitchellh/copystructure v1.0.0 // indirect
-	github.com/stretchr/testify v1.4.0 // indirect
-	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17 // indirect
+	github.com/openlyinc/openlynt v0.0.1
```
Comment (bdimcheff) [inline go.mod:8]: should this be the vanity url?

File: go.mod [hunk 2]
```diff
@@ -1,17 +1,10 @@
-module github.com/openlyinc/openlynt
+module go.openly.dev/openlynt
 
 go 1.13
```
Comment (bdimcheff) [inline go.mod]: this is soooo old, i'd bump it to at least 1.19. modules work differently with old versions.

File: go.mod (hunk 1 above)
Comment (mwielbut) [reply go.mod:8]: That's the problem, until I publish this I can't update this dependency. Unless I'm missing something...

File: go.mod (hunk 2 above)
Comment (mwielbut) [reply go.mod]: Good call. Done. I also updated goutils to address a security issue.

File: go.mod (hunk 1 above)
Comment (bdimcheff) [reply go.mod:8]: ahhh ok yeah that makes sense

**Review by bdimcheff — APPROVED** [review]

**Pull request merged**